File: Harvard cs50 course/Simple Mario Less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int a;
    do
    {
        a = get_int("Height: ");
    }
    while (a < 1 || a > 8); //do-while loop
    for (int i = 0; i < a; i++) //for loop
    {
        for (int j = 0; j < a - i - 1; j++)
        {
            printf(" "); //printing
        }
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }
        printf("\n"); //new line
    }
}

File: Harvard cs50 course/Credit/mario.py
import cs50
while True:
    a = get_int("Height: ")
    if a >= 1 and a <= 8:
        break

for i in range(a):
    for j in range(a-i-1):
        print(" ", end="")
    for j in range(i+1):
        print("#", end="")
    print("  ", end="")
    for j in range(i+1):
        print("#", end="")
    print()